src/components/Reports/ReportView.tsx

```diff
@@ -97,7 +97,27 @@ export default function ReportView(): JSX.Element {
           <Typography fontSize='24px' fontWeight={600}>
             {report ? `Report (${report?.year}-Q${report?.quarter})` : ''}
           </Typography>
-          {report?.status !== 'Submitted' && <Button label={strings.REPORT_EDIT} icon='iconEdit' onClick={startEdit} />}
+          <Box gap={theme.spacing(1)}>
+            {report?.isAnnual &&
+              (showAnnual ? (
+                <Button
+                  label={strings.REPORT_VIEW_QUARTERLY}
+                  type='productive'
+                  priority='secondary'
+                  onClick={() => switchPages(false)}
```

Comment:
unrelated to this PR, thoughts on using a type as an argument in the switchPages ('quarterly' | 'annual')..

Reply:
That's a nice upgrade 👍